chemprop/cli/hpopt.py

```diff
@@ -242,6 +242,9 @@ def update_args_with_config(args: Namespace, config: dict) -> Namespace:
             case "init_lr_ratio":
                 setattr(args, "init_lr", value * args.max_lr)
 
+            case "batch_size":
+                setattr(args, "batch_size", int(value))
```

Comment:
Whatever `tune` method we call should return an `int`, making this cast unnecessary?

Reply:
Also mentioned here by @KnathanM: https://github.com/chemprop/chemprop/pull/873#discussion_r1608761150

Reply:
Yes, good catch.
Working on it

Reply:
@akshatzalte I think this hasn't been addressed yet. As `tune.choice(np.logspace(start=4, stop=8, num=5, base=2, dtype=int))` is used, `batch_size` should always be an integer. The code here can be removed.